modules/installing-the-odh-operator-v1.adoc

```diff
@@ -0,0 +1,37 @@
+:_module-type: PROCEDURE
+//pv2hash: 9cbc09e0-cac2-4eb3-8f30-09e6469c5164
+
+[id='installing-the-odh-operator-v1_{context}']
+= Installing the Open Data Hub Operator (v1)
+
+.Prerequisites
+* You are using OpenShift Container Platform 4.11 or later.
+* Your OpenShift cluster has a minimum of 16 CPUs and 32GB of memory across all OpenShift worker nodes.
+* You can log in as a user with `cluster-admin` privileges.
+
+.Procedure
+. Log in to the OpenShift Container Platform as a user with `cluster-admin` privileges. For a developer installation from link:http://try.openshift.com[try.openshift.com], you can log in as the `kubeadmin` user.
+. Select *Operators* -> *OperatorHub*.
+. In the *Filter by keyword* field, enter `Open Data Hub Operator`.
+. Click the *Open Data Hub Operator* tile.
+. If the *Show community Operator* window displays, read the information and then click *Continue*.
```

Comment:
_Displays_ is a transitive verb that requires a subject. Use _appears_ instead.

Reply:
IBM style guide suggest "opens" instead of "displays" or "appears"

Reply:
Thank you - I should've thought to check that one myself. Might be another old recommendation I 'm still carrying around :D

On a lighter note: The style guide for an old company I worked said to use _appears dimmed_ instead of _grayed out_. I always thought that was such an awful recommendation :)